internal/bundle/plain.go

```diff
@@ -0,0 +1,66 @@
+package bundle
```

Comment:
This might be a good candidate for it's own PR given we're moving around the getObjects function here.